src/pages/Main/Report.js

```diff
@@ -1,17 +1,19 @@
 import React from "react";
-//import VolunteerReport from "../Reports/VolunteerReport";
+import VolunteerReport from "../Reports/VolunteerReport";
 import KindReport from "../Reports/KindReport";
-import AppealReport from "../Reports/AppealReport";
-import RequestReport from '../Reports/RequestReport';
+// import AppealReport from "../Reports/AppealReport";
```

Comment:
Why was appeal reports commented out?

Reply:
This was asked to be commented out by @rtkanan in one of the meetings. If need it can be enabled. So I am not sure.